src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -67,15 +68,16 @@ def __init__(self):
           continue
         self.trials[config['app_args']] = config['probability']
     except Exception:
-      logs.log_warn('Unable to parse config file because of the following error:')
+      logs.log_warn("""
+      Unable to parse config file because of the following error:
+      """)
```

Comment:
nit: Please keep using single quotes. Shorten sentence due to redundancy?
'Unable to parse config file:'

Reply:
done